src/lib/index.ts

```diff
@@ -45,6 +45,14 @@ export { ErrorPage500 } from './components/error-pages/ErrorPage500.component';
 export { ErrorPageAuth } from './components/error-pages/ErrorPageAuth.component';
 export { TextBadge } from './components/textbadge/TextBadge.component';
 export { SpacedBox } from './components/spacedbox/SpacedBox';
+export {
+  Layout as Layout2,
+  ContextContainer,
+  OverallSummary,
+  MainContent,
+} from './components/layout/v2';
+export { TwoPanelLayout } from './components/layout/v2/panels';
+export { AppContainer } from './components/layout/v2/AppContainer';
```

Comment:
In terms of code architect/folder, I suggest we could put all those components in one folder, so when we import from other project, all the component will come from one place.